CIScripts/Tests/TCPoMPLSoGRETest.ps1

```diff
@@ -0,0 +1,40 @@
+function Test-TCPoMPLSoGRE {
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session1,
+           [Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session2,
+           [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+
+    Write-Host "Running TCP Over MPLS Over GRE test"
+
+    Initialize-TestConfiguration -Session $Session1 -TestConfiguration $TestConfiguration
+    Initialize-TestConfiguration -Session $Session2 -TestConfiguration $TestConfiguration
+
+    Write-Host "Running containers"
+    $NetworkName = $TestConfiguration.DockerDriverConfiguration.NetworkConfiguration.NetworkName
+    $ServerID = Invoke-Command -Session $Session1 -ScriptBlock { docker run --network $Using:NetworkName -d iis-tcptest }
+    $ClientID = Invoke-Command -Session $Session2 -ScriptBlock { docker run --network $Using:NetworkName -d microsoft/nanoserver ping -t localhost }
```

Comment:
Why are we pinging localhost in a TCP test? I think it should be replaced by some other command that doesn't run continuously